scripts/publish.sh

```diff
@@ -28,5 +28,28 @@ echo "$NPM_REGISTRY_NO_PROTO/:_authToken=\"$NPM_TOKEN\"" > $NPMRC_PATH
 
 # cd to the package and publish it!
 PKG_PATH=`yarn workspaces info -s | jq -r ".[\"$PKG_NAME\"].location"`
-cd $PKG_PATH
+pushd $PKG_PATH > /dev/null
 npm publish --registry $NPM_REGISTRY
+popd > /dev/null
+
+# === BEGIN Boilerplate Rebuild ===
+
+if [[ "$PKG_NAME" == "zapier-platform-core" ]]; then
+    LEGACY_VERSION=$(curl https://registry.npmjs.org/zapier-platform-legacy-scripting-runner/latest | jq -r .version)
+    CORE_VERSION=$(cat $PKG_PATH/package.json | jq -r .version)
+fi
+
+if [[ "$PKG_NAME" == "zapier-platform-legacy-scripting-runner" ]]; then
+    CORE_VERSION=$(curl https://registry.npmjs.org/zapier-platform-core/latest | jq -r .version)
+    LEGACY_VERSION=$(cat $PKG_PATH/package.json | jq -r .version)
+fi
+
+if [[ "$CORE_VERSION" != "" && "$LEGACY_VERSION" != "" ]]; then
+    # Build boilerplate and let Zapier know about this
+    ./boilerplate/scripts/build.sh $CORE_VERSION $LEGACY_VERSION &&
+    ./boilerplate/scripts/upload.sh $CORE_VERSION &&
+    ZIP_HASH=$(sha1sum "./boilerplate/build/$CORE_VERSION.zip" | awk '{print $1}') &&
+    curl -H 'content-type:application/json' -d "{\"version\":\"$CORE_VERSION\",\"id\":\"$ZIP_HASH\"}" $BOILERPLATE_UPDATE_URL | jq .status
```

Comment:
`BOILERPLATE_UPDATE_URL` has been configured on Travis. It's a hook to which we can post requests to notify Zapier to update any internal resources.